iOS_SDK/OneSignalSDK/build_all_frameworks.sh

```diff
@@ -24,6 +24,8 @@ create_xcframework() {
     echo "Created ${FRAMEWORK_FOLDER_NAME}"
     echo "Archiving ${FRAMEWORK_NAME}"
 
+    xcodebuild -list
```

Comment:
Is this just to help debug if something goes wrong?

Reply:
It's copied over from the file on main: https://github.com/OneSignal/OneSignal-iOS-SDK/pull/1340/files
However it may be leftover from the testing process.